src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/PrisonIepLevel.kt

```diff
@@ -15,11 +15,13 @@ import org.hibernate.annotations.JoinColumnsOrFormulas
 import org.hibernate.annotations.JoinFormula
 import org.hibernate.type.YesNoConverter
 import java.io.Serializable
+import java.math.BigDecimal
+import java.time.LocalDate
 
 @Entity
-@IdClass(AvailablePrisonIepLevel.Companion.PK::class)
+@IdClass(PrisonIepLevel.Companion.PK::class)
 @Table(name = "IEP_LEVELS")
-data class AvailablePrisonIepLevel(
+data class PrisonIepLevel(
 
   @Id
   @Column(name = "IEP_LEVEL")
```

Comment:
A real shame this attribute is called `id` given it is just part of the real ID. I had the read the HQL below several times before I understood what was going on.

Any chance it could be renamed in a separate pr?

Reply:
Yep sure